ohm-js/test/test-ohm-syntax.js

```diff
@@ -976,6 +976,47 @@ test('inheritance', t => {
   t.end();
 });
 
+test('override with "..."', t => {
+  let g = ohm.grammar('G { letter := "@" | ... }');
+  t.equal(g.match('@', 'letter').succeeded(), true);
+  t.equal(g.match('a', 'letter').succeeded(), true);
+
+  g = ohm.grammar('G { letter := ... | "@" }');
+  t.equal(g.match('@', 'letter').succeeded(), true);
+  t.equal(g.match('a', 'letter').succeeded(), true);
+
+  g = ohm.grammar('G { letter := "3" | ... | "@" }');
+  t.equal(g.match('@', 'letter').succeeded(), true);
+  t.equal(g.match('3', 'letter').succeeded(), true);
+
+  t.ok(ohm.grammar('G { letter := ... }'), 'it allows `...` as the whole body');
+
+  t.throws(
+      () => ohm.grammar('G { foo = ... }'),
+      /Expected "}"/,
+      "it's not allowed in a rule definition");
+
+  t.throws(
+      () => ohm.grammar('G { letter += ... }'),
+      /Expected "}"/,
+      "it's not allowed when extending");
+
+  t.throws(
+      () => ohm.grammar('G { letter := "@" "#" | ... }'),
+      /inconsistent arity/,
+      'it ensures consistent arity');
+
+  /*
+    TODO:
```

Comment:
I suspect the grammar will break if someone uses more than one `...` in an override body. It would be good to catch this and display a nice error message.

Reply:
Good catch. I had a check but it must have disappeared in refactoring.